packages/common/src/visualizations/CartesianChartDataModel.ts

```diff
@@ -257,10 +274,14 @@ export type CartesianChartDisplay = {
         position?: string;
         format?: Format;
     }[];
-    series?: Record<
```

Comment:
I had to change this Record to be able to get the colors on the backend, I think this is related with tsoa not liking Record types and removing some content in the payload